components/sections/ProjectSection.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState, useRef } from "react";
+import Image from "next/image";
+import Wrapper from "../wrappers/Wrapper";
+import arrow from "../../public/assets/images/arrow.svg";
```

Comment:
You don't need to go out this  `../../public/assets/images/arrow.svg` since it is in public you can do it the other way